.github/workflows/greetings.yml

```diff
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+    greeting:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/first-interaction@v1
+                with:
+                    repo-token: ${{ secrets.GITHUB_TOKEN }}
+                    issue-message: '**Thanks for opening your first issue here!**<br/><br/>Be sure to follow the issue template! Your issue will be reviewed by a maintainer and labeled accordingly for further actions.'
+                    pr-message: '**Thanks for opening this pull request!**<br/><br/>Please check out our [contributor checklist](https://docs.bisq.network/contributor-checklist.html) and have a look if issues that are found by *Travis* or *Codacy* running basic checks on your PR. Also make sure your commits are signed and you applied the Bisq code style and formatting.<br/><br/>A maintainer will add an `is:priority` label to your PR if it is up for compensation. Please checkout our [blog post](https://bisq.network/blog/TBD/) for more details on this.'
```

Comment:
```suggestion
                    pr-message: '**Thanks for opening this pull request!**<br/><br/>Please check out our [contributor checklist](https://docs.bisq.network/contributor-checklist.html) and have a look if *Travis* or *Codacy* found any issues with your PR. Also make sure your commits are signed, and that you applied Bisq code style and formatting.<br/><br/>A maintainer will add an `is:priority` label to your PR if it is up for compensation. Please see our [blog post](https://bisq.network/blog/TBD/) for more details on this.'
```

Basic edits.

Reply:
Also, maybe link "Bisq code style and formatting" here:
https://github.com/bisq-network/style/issues

Reply:
Thanks! Just force pushed the changes.